src/metabase/driver/exasol.clj

```diff
@@ -18,6 +19,28 @@
             [metabase.util.i18n :refer [trs]]
             [java-time :as t]))
 
+
+(defn get-jdbc-driver-version []
+  (com.exasol.jdbc.EXADriver/getVersionInfo))
+
+(defn get-driver-version
+  ([]
+   (get-driver-version "META-INF/maven/metabase/exasol-driver/pom.properties"))
+  ([resource]
+   (when-let [url ^java.net.URL (io/resource resource)]
+     (with-open [stream (.openStream url)]
+       (let [p (java.util.Properties.)]
+         (try
+           (.load p stream)
+           (.getProperty p "version")
+           (catch Exception _)))))))
+
+(defn- log-driver-version []
+  (log/info (u/format-color 'green (format "Loading Exasol Metabase driver %s, Exasol JDBC driver: %s"
```

Comment:
is it a good practice to use colors in logs?

Reply:
The Metabase log is very colorful:
<img width="1108" alt="image" src="https://user-images.githubusercontent.com/4711730/149959335-c8ee8197-e5bc-4dcf-8773-e58540db43c3.png">